source/h2_stream.c

```diff
@@ -346,10 +382,46 @@ static void s_stream_cross_thread_work_task(struct aws_channel_task *task, void
     aws_http_stream_release(&stream->base);
 }
 
+static void s_stream_data_write_destroy(
+    struct aws_h2_stream *stream,
+    struct aws_h2_stream_data_write *write,
+    int error_code) {
+
+    AWS_PRECONDITION(stream);
+    AWS_PRECONDITION(write);
+    if (write->on_complete) {
+        write->on_complete(&stream->base, error_code, write->user_data);
+    }
+    if (write->data_stream) {
+        aws_input_stream_release(write->data_stream);
+    }
+    aws_mem_release(stream->base.alloc, write);
+}
+
+static void s_h2_stream_destroy_pending_writes(struct aws_h2_stream *stream, int error_code) {
```

Comment:
s_h2_stream_destroy_pending_writes() doesn't need an error_code param, the rest of the streams will all complete with `AWS_ERROR_HTTP_STREAM_HAS_COMPLETED`